.github/workflows/release.yml

```diff
@@ -24,15 +24,16 @@ jobs:
   build:
     needs: create_release
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu22.04-aarch64, ubuntu22.04, ubuntu20.04-aarch64, ubuntu20.04, ubuntu18.04]
-    runs-on: macos-13
+        os: [ubuntu24.04, ubuntu24.04-aarch64, ubuntu22.04, ubuntu22.04-aarch64, ubuntu20.04, ubuntu20.04-aarch64, fedora39, archlinux]
```

Comment:
can we include `fedora39-aarch64` and `archlinux-aarch64` here as well? I'll upload those manually to the darwin-tools repo